torch/linalg/__init__.py

```diff
@@ -140,6 +140,110 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+eigh = _add_docstr(_linalg.linalg_eigh, r"""
+linalg.eigh(input, UPLO='L') -> tuple(Tensor, Tensor)
+
+This function returns eigenvalues and eigenvectors
+of a complex Hermitian (conjugate symmetric) or real symmetric matrix :attr:`input`
+represented by a namedtuple (eigenvalues, eigenvectors).
+
+This function calculates all eigenvalues (and vectors) of :attr:`input`
+such that :math:`\text{input} = V \text{diag}(e) V^H`, where :math:`e` is eigenvalues tensor
```

Comment:
"eigenvalues tensor" -> "tensor of eigenvalues"
"eigenvectors tensor" -> "tensor of eigenvectors"

Reply:
This second paragraph is mostly redundant with the first. Can they be combined?

Reply:
UPLO should be mentioned here and in the mathematics, too.

Reply:
Changed now to
```
This function computes the eigenvalues and eigenvectors
of a complex Hermitian (or real symmetric) matrix :attr:`input` or batches of such matrices
such that :math:`\text{input} = V \text{diag}(w) V^H`, where :math:`w` is the tensor of eigenvalues,
:math:`V` is the tensor of eigenvectors and :math:`^H` is the conjugate transpose operation.

Since :attr:`input` is assumed to be consisting of Hermitian matrices,
only the lower triangular portion of these matrices is used by default (``UPLO = 'L'``)  in computations
and the imaginary part of the diagonal will always be treated as zero.
```